### ISSUE TITLE
probleme index.php

### ISSUE THREAD

**Issue by ludo77** [broncowdd/googol#35]
Bonjour,

j'ai installé googol sur mon serveur, quand je lance une recherche j'ai une erreur:

 Warning: file_get_contents(INACTIVE.ini): failed to open stream: No such file or directory in /var/www/googol/index.php on line 126

et quand je fais une recherche image j'ai la meme erreur et en plus pas de résultat pour "ma_recherche"

et pour video j'ai la meme erreur warning

Merci

**Comment by broncowdd** [comment]
En effet, je viens de voir qu'il y a un souci de ce côté: je pense qu'il
s'agit d'un changement de la part de google, un genre de contremesure...
il faudra que je regarde ça rapidement. Merci du retour ;-)

**Comment by broncowdd** [comment]
Heu, en fait, c'est de ma faute ^^: un test sur l'existence d'un fichier
avant de le charger... je corrige ça.
Si tu veux corriger de suite, tu peux changer la fonction still_pause
(ligne 125 d'index.php)
par celle-ci:
    function still_pause(){
        if (is_file('INACTIVE.ini')){
            $since=intval(file_get_contents('INACTIVE.ini'));
        }else{$since=0;}
        //echo 'depuis '.$since.'// pendant '.PAUSE_DURATION.' min// il
est '.date('U').'// jusqua '.($since+(PAUSE_DURATION*60));
        return $since+(PAUSE_DURATION*60)>date('U');
    }
A +
Bronco

**Comment by ludo77** [comment]
merci

je vais tester cela

**Comment by ludo77** [comment]
j'ai de ce fait une autre erreur qui est apparu:

quand je fais une recherche image, j'ai un message pas de resultat pour **ma_recherche**

**Comment by broncowdd** [comment]
ça peut venir de google: ils changent la structure de leurs pages de  temps à autres et il faut parfois revoir la regex pour les images.
Ceci dit, ça ne semble pas être le cas ici: mon googol fonctionne toujours: http://googol.warriordudimanche.net 

Y a-t-il une erreur dans les logs ?

**Comment by ludo77** [comment]
je n'ai pas vu d'erreur dans les logs

**Comment by ludo77** [comment]
j'ai reinstallé googol en faisant un wget, puis un unzip et pour finir un chown .

Et j'ai toujours le probleme pour la recherche des images

**Comment by ludo77** [comment]
Bonjour,
je viens de reinstaller googol, et je vois que la recherche image ne fonctionne pas, j'ai donc tester :http://googol.warriordudimanche.net.
J'ai la même erreur qui apparait.

**Comment by broncowdd** [comment]
En effet, google change régulièrement la structure de la page, du 
coup, je dois adapter les regexes... Je regarde asap ;-) 
 Merci


Le mer. 20 avril 2016 à 10:49, ludo77 <notifications@github.com> a 
écrit :

**Comment by Chouchen** [comment]
La recherche d'image semble fonctionner autant chez moi que sur http://googol.warriordudimanche.net

**Comment by jerrywham** [comment]
Oui, ça fonctionne depuis la dernière mise à jour de l'ami Bronco.
